### PR TITLE
docs(install): Easier way to install using omz

### PR DESCRIPTION
**Pull request by gavindsouza**
### Changes

* Use omz command to enable plugin
* ~~Updated git command to do a shallow clone to save space~~

### CODE REVIEW

**Comment by phy1729** [comment]
The changes to step 2 seem fine. For the changes to step 1, shallow clones aren't preferred cf. #855.

**Comment by gavindsouza** [comment]
> shallow clones aren't preferred cf. #855.

Thanks for this! Updated the PR.

File: INSTALL.md [hunk 1]
```diff
@@ -83,13 +83,12 @@ your `.zshrc`.
     git clone https://github.com/zsh-users/zsh-syntax-highlighting.git ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-syntax-highlighting
     ```
 
-2. Activate the plugin in `~/.zshrc`:
+2. Activate the plugin:
 
     ```zsh
-    plugins=( [plugins...] zsh-syntax-highlighting)
+    omz plugin enable zsh-syntax-highlighting
```
Comment (danielshahaf) [inline INSTALL.md:89]: Is this syntax supported by all oh-my-zsh versions run by users who'll be reading this file?

Comment (danielshahaf) [inline INSTALL.md:89]: Does the proposed method make zsh-syntax-highlighting run after all other plugins?  The incumbent method does.

**Pull request closed without merging**